spec/src/main/asciidoc/converters.asciidoc

```diff
@@ -96,7 +87,8 @@ e.g. With this config `myPets=dog,cat,dog\\,cat`, the values as an array will be
 
 If no built-in nor custom `Converter` for a requested Type `T`, an implicit Converter is automatically provided if the following conditions are met:
 
-* The target type `T` has a public Constructor with a String parameter, or
-* the target type `T` has a `public static T valueOf(String)` method, or
-* the target type `T` has a `public static T parse(CharSequence)` method
+* The target type {@code T} has a {@code public static T of(String)} method, or
```

Comment:
Is it worth pointing out that implementations can provide additional support (e.g. `URI.create`) that are functionally equivalent, or perhaps add new features on top of the required?

Reply:
For what purpose? The change is making sure the provided converters are exactly same among implementations so that the app is portable among app servers.